### ISSUE TITLE
Epub Download?

### ISSUE THREAD

**Issue by Eddie-Hartman** [braydie/HowToBeAProgrammer#127]
How do I actually get the epub version to download? The gitbook link requires an account AND a phone number. Is that the only option?

**Comment by nickkelley42** [comment]
I agree that an option for downloading and/or making the epub without creating an account would be nice. What I ended up doing was using [Pandoc](https://pandoc.org/epub.html) to create the epub myself.

Inside the directory for the language you want, run:

```
pandoc --metadata title='How To Be A Programmer' -o HowToBeAProgrammer.epub $(find -name '*.md' | sort)
```

It's a bit gnarly because `sort` puts the READMEs after the chapters, but it's good enough for me.

**Comment by braydie** [comment]
Hey! Yeah GitBook has changed it's format and unfortunately the links are
broken. Ideally I'd like to get an ePub/PDF version bundled with the
markdown files. I'd love some help getting this sorted as my time is
limited..!

**Comment by nickkelley42** [comment]
I can make a PR with a cleaned-up version of the command I used to generate the epub. My only reservation is my epub/PDF knowledge is pretty limited; the links in the ebook that I've made are broken, and I'm not sure of a clean way to fix the links without breaking the nice structure that exists in the GitHub repository.
